File: src/commands/invite.js
module.exports = exports = {
  name: 'invite',
  usage: '',
  enabled: true,
  /**
   * @param {import('discord.js').Message} message
   * @param {import('../index.js').GLOBALS} GLOBALS
   */
  process: async (message, GLOBALS) => {
    const embed = new GLOBALS.Embed()
      .setTitle('Add me to your server!')
      .setURL(`https://discord.com/oauth2/authorize?client_id=${GLOBALS.client.user.id}&scope=bot&permissions=285355008 `)
    message.reply(embed)
  }
}

File: commands/invite.go
package commands

import "github.com/bwmarrin/discordgo"

func init() {
	AddCommand(&discordgo.ApplicationCommand{
		Name:        "invite",
		Description: "Invite ScrimBot to another server",
	}, func(s *discordgo.Session, i *discordgo.InteractionCreate) {
		s.InteractionRespond(i.Interaction, &discordgo.InteractionResponse{
			Type: discordgo.InteractionResponseChannelMessageWithSource,
			Data: &discordgo.InteractionResponseData{
				Content: "go here to add scrimbot https://discord.com/oauth2/authorize?client_id=715030981894995998&scope=bot&permissions=2432904272",
			},
		})
	})
}
